### ISSUE TITLE
Course cards use the lang attribute incorrectly

### ISSUE THREAD

**Issue by Mankro** [apluslms/a-plus#783]
Course cards use the `lang` HTML attribute incorrectly at least for multilingual courses. The value may not contain multiple languages. The WAWE browser plugin shows accessibility errors. Only one language is allowed and I assume it should be the language that is used in the content of the element that has the lang attribute. This should be checked.

It is a challenge that the course card may have content in multiple languages. At least teachers may see the "hidden from students" label in the language used in Django translations, but the front page may use a different language than the language of the course. In multilingual courses, the course title is usually only given in English, but the user may have selected another language.

Course cards are used in the A+ front page and the new course instances list http://localhost:8000/def/instances/ (from #772).
